osc-service-api/src/main/java/org/osc/core/broker/service/dto/SecurityGroupInterfaceDto.java

```diff
@@ -39,10 +42,7 @@
     private String name;
 
     @ApiModelProperty(required = true)
```

Comment:
false since this is optional in some cases right when policy mapping is not supported